src/ocamlorg_frontend/pages/jobs.eml

```diff
@@ -72,5 +72,6 @@ OCaml community."
             <% ); %>
         </div>
         <% ); %>
+       If you want to contribute a job post, check out this <a href="https://github.com/ocaml/ocaml.org/blob/main/CONTRIBUTING.md#jobs" class="no-underline hover:underline font-black" target="_blank">contributing guide</a> on GitHub.
```

Comment:
It will be better to wrap this text in a `<div class="prose">...</div>` and removing the class from the `a` element, so that it uses the styles coming from Tailwind Typography plugin which are used everywhere else on the site.

Reply:
great, thanks!